standalone-node/cluster_installers/docs/standalone-edge-node-installer.md

```diff
@@ -55,48 +53,42 @@ There is two options to build a package
 
 To install Microvisor on the Standalone Node
 
-1. Copy the package to a writable directory ie. `/tmp/rke2-artifacts` 
+1. Copy the package to a writable directory ie. `/tmp/k3s-artifacts` 
 
 ```shell
-mkdir /tmp/rke2-artifacts
-cp sen-rke2-package.tar.gz /tmp/rke2-artifacts
+mkdir /tmp/k3s-artifacts
+cp sen-k3s-package.tar.gz /tmp/k3s-artifacts
 ```
 
 2. Unpack the package
 
 ```shell
-cd /tmp/rke2-artifacts
-tar xf sen-rke2-package.tar.gz
+cd /tmp/k3s-artifacts
+tar xf sen-k3s-package.tar.gz
 ```
 
 3. Run installer
 
-- By default installer is expecting the packages in `/tmp/rke2-artifacts`
+- By default installer is expecting the packages in `/tmp/k3s-artifacts`
 
 ```shell
-./sen-rke2-installer.sh
+./sen-k3s-installer.sh
 ```
 
 - If different path is selected to download the artifacts to then the installer can be pointed to it by providing the path as an argument
 
 ```shell
-./sen-rke2-installer.sh /some/other/directory
+./sen-k3s-installer.sh /some/other/directory
 ```
 
 1. Wait for install to finish and then all pods to come up running
 
 ```shell
-sudo -E KUBECONFIG=/etc/rancher/rke2/rke2.yaml /var/lib/rancher/rke2/bin/kubectl get pods -A
+sudo -E KUBECONFIG=/etc/rancher/k3s/k3s.yaml /usr/bin/k3s kubectl get pods -A
 ```
+TODO: move to /usr/local/bin/ when binaries are available.
 
-## Uninstalling
-
```

Comment:
if we are removing uninstall section and if we don't support it, then we should have a section to reset the cluster and re-initialize. Also say if the cluster is instable after re-initialization, then re-install might be needed.